docs/ADR-22-quests-progress-ui.md

```diff
@@ -0,0 +1,98 @@
+# 2021-01-19 - Quests Progress UI
+
+## Context and Problem Statement
+
+How to render the Quests progression?
+
+Based on the context of a previous meeting with Pravus, Alex, Pablo, Marcosnc and Mendez on Jan 7, 2021:
+
+## Portable experiences & quests
+
+### How do we integrate quests with portable experiences?
+
+- Unity should uniformly render all quests' progression in the same way.
+
+- Every quest is a regular scene that runs in a "global" scene. "global" means without limits of parcels.
+
+#### The quest (the SDK scene running the logic and visible part of the quest)
+
+- The quest' scene will use an SDK `QuestTrackingInformation(steps & progression)` primitive to consistently render the quest UI
+
+- The quest' scene will handle the execution of the logic of the quest
+
+- The progression of the quest will have to be saved and loaded from the quest'scene
```

Comment:
A bit of clarification on "saved and loaded"? The quests persistence isn't going to be handled by the server?

Reply:
That was the raw notes of the previous meeting. I can't recall correctly that part

Reply:
That part means that the scene, using our SDK, would read-write the quest progress, similar to using a regular game achievements API to report progress.